File: python/485_Max_Consecutive_Ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        maxCount = 0
        tempCount = 0
            
        for num in nums:
            if (num == 0): 
                maxCount = max(maxCount, tempCount)
                tempCount = 0
            else:
                tempCount += 1
        maxCount = max(maxCount, tempCount)
        
        return maxCount

File: cs/485_Max_Consecutive_Ones.cs
public class Solution {
    public int FindMaxConsecutiveOnes(int[] nums) {
        int maxCount = 0;
        int tempCount = 0;
        
        foreach (int num in nums) {
            if (num == 0) {
                maxCount = Math.Max(maxCount, tempCount);
                tempCount = 0;
            } else {
                tempCount++;
            }
        }
        maxCount = Math.Max(maxCount, tempCount);
        
        return maxCount;
    }
}
